### PR TITLE
issue #7 - whats ansible

### PR DESCRIPTION
**Pull request by dj4ngo**

### CODE REVIEW

File: _drafts/contribute-to-ansible.markdown [hunk 1]
```diff
@@ -16,9 +16,32 @@
...
-    petit exemple d'installation d'un paquet (liens vers le site officiel de ansible)
-    Différencier serveur de client (machine hote ou cible) retrouve rles vrais termes ansible
+    
+If you are reading this article, you should propably be already familiar with Ansible... So if you're not reading it, we'll explain it quickly ...
+    
+Why Ansible ? Because it's cool! But not only ... It's really really more practical and easy to use than just cool. Ansible will help you to manage easily your laptops/servers/network devices/IoT/everything or almost ! Ansible force lie in the capability to easily configure a lot of devices and guaranty that the configuration is consistent in all targets regardless the initial state. How is it done ? The concept behind an Ansible module is to be re-enterable!
+
+OK, that sounds great, but what's the price? Ansible is an open-source project which have a growing community and it is really easy to use, so it won't costs many times (it will even save so much !) and it's fully free (you can have some support on some modules for profesionnal purposes regarding your use cases).
+
+####Principle :
```
Comment (4383) [inline _drafts/contribute-to-ansible.markdown]: place space between titles and markdow syntaxe

File: _drafts/contribute-to-ansible.markdown [hunk 2]
```diff
@@ -16,9 +16,32 @@
...
+
+To define what to perform on managed nodes you must creates playbook. This is still YAML files. You can find some documentation about playbooks [here](http://docs.ansible.com/ansible/latest/playbooks.html). To avoid having code duplication and win some flexibility I strongly recommend you to use [playbooks roles](http://docs.ansible.com/ansible/latest/playbooks_roles.html).
+
+We recommend to follow the [best practices](http://docs.ansible.com/ansible/latest/playbooks_best_practices.html) to organize your Ansible directory layout
+
+
+Get more informations on [how it works](https://www.ansible.com/how-ansible-works) or more generaly on [Ansible official site](https://www.ansible.com/) 
+The [Ansible documentation](http://docs.ansible.com/ansible/latest/index.html) is really up-to-date, well organised and contained a lots of example. It will be a great help for you. 
+
+
```
Comment (4383) [inline _drafts/contribute-to-ansible.markdown:48]: Talk about DevOps! Ansible is greate DevOps tools so what's ansible must introduce this theme

**Review by 4383 — CHANGES_REQUESTED** [review]

File: _drafts/contribute-to-ansible.markdown [hunk 3]
```diff
@@ -16,9 +16,32 @@
     git installé
 
 ## What's Ansible?
-    Principe de fonctionnement
-    petit exemple d'installation d'un paquet (liens vers le site officiel de ansible)
-    Différencier serveur de client (machine hote ou cible) retrouve rles vrais termes ansible
+    
+If you are reading this article, you should propably be already familiar with Ansible... So if you're not reading it, we'll explain it quickly ...
+    
+Why Ansible ? Because it's cool! But not only ... It's really really more practical and easy to use than just cool. Ansible will help you to manage easily your laptops/servers/network devices/IoT/everything or almost ! Ansible force lie in the capability to easily configure a lot of devices and guaranty that the configuration is consistent in all targets regardless the initial state. How is it done ? The concept behind an Ansible module is to be re-enterable!
```
Comment (4383) [inline _drafts/contribute-to-ansible.markdown]: Why Ansible?
But not only...
almost!
done?

Comment (4383) [reply _drafts/contribute-to-ansible.markdown]: Talk about provisionning tools and make a short compare with other solutions like puppet/chef

**Review by 4383 — CHANGES_REQUESTED** [review]

**Comment by 4383** [comment]
do you have review my comments?

**Comment by 4383** [comment]
[ponctuation](https://fr.wikihow.com/employer-correctement-la-ponctuation-en-anglais)

File: _drafts/contribute-to-ansible.markdown (hunk 3 above)
Comment (dj4ngo) [reply _drafts/contribute-to-ansible.markdown]: provisionning tools ? What do you think about ?

Comment (4383) [reply _drafts/contribute-to-ansible.markdown]: make an quickly introduce to the provisionning tools concept.
Ansible as an provisionnig tools. Provisionnig tools allow to you to manage simply your machine maintenance. With these tools you can drive machine management in an industrial way.

Others provisionning tools exist, like puppet / chef but their are out of the scope of this post.

**Comment by dj4ngo** [comment]
cc @4383, modifications dones!

**Review by 4383 — APPROVED** [review]
Excellent!

**Pull request merged**